http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20230713_app_controlled_resource_constraints.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="dvoet" id="drop_IDX_APP_CONTROLLED_RESOURCE_UNIQUE">
+        <dropUniqueConstraint constraintName="IDX_APP_CONTROLLED_RESOURCE_UNIQUE" tableName="APP_CONTROLLED_RESOURCE"/>
+    </changeSet>
+    <changeSet logicalFilePath="leonardo" author="dvoet" id="APP_CONTROLLED_RESOURCE_PK">
+        <addPrimaryKey columnNames="resourceId" tableName="APP_CONTROLLED_RESOURCE"/>
```

Comment:
resourceId is a UUID and should not be shared between apps so using it as a PK on this table works